pkg/destroy/aws/aws.go

```diff
@@ -996,6 +998,30 @@ func deleteEC2NATGatewaysByVPC(ctx context.Context, client *ec2.EC2, vpc string,
 	return err
 }
 
+func deleteEC2PlacementGroup(ctx context.Context, client *ec2.EC2, id string, logger logrus.FieldLogger) error {
+	response, err := client.DescribePlacementGroupsWithContext(ctx, &ec2.DescribePlacementGroupsInput{
+		GroupIds: []*string{aws.String(id)},
+	})
+	if err != nil {
+		if err != nil {
+			if err.(awserr.Error).Code() == "InvalidPlacementGroup.Unknown" {
```

Comment:
It is terribly unfortunate that we have to use these magic string values to determine whether a resource has been deleted. It would be so much nicer if the AWS SDK exposed in its public interface the HTTP status code. There is nothing to do here, it is just a gripe.

Reply:
My comment may not have loaded while you reviewed, but I made a note as well about where this came from, I think I got the correct magic string https://github.com/openshift/installer/pull/5528#discussion_r781371314

Reply:
Yep. I also verified that it was correct by trying to delete a placement group that I had already deleted.